gprofiler/main.py

```diff
@@ -662,13 +703,15 @@ def main():
             args.python_mode,
             runtimes,
             client,
+            args.collect_metrics,
             state,
             not args.disable_container_names,
             remote_logs_handler,
             args.php_process_filter,
         )
         logger.info("gProfiler initialized and ready to start profiling")
-
```

Comment:
Bring it back

Reply:
![image](https://user-images.githubusercontent.com/2903914/126071542-029c9ff5-8699-411a-8b56-1d557bc7089d.png)